engine/installation/linux/ubuntulinux.md

```diff
@@ -14,32 +14,31 @@ Docker is supported on these Ubuntu operating systems:
 - Ubuntu Trusty 14.04 (LTS)
 - Ubuntu Precise 12.04 (LTS)
 
-This page instructs you to install using Docker-managed release packages and
-installation mechanisms. Using these packages ensures you get the latest official
+This page instructs you to install Docker on Ubuntu, using packages provided by
+Docker. Using these packages ensures you get the latest official
 release of Docker. If you are required to install using Ubuntu-managed packages,
-consult the Ubuntu documentation.
+consult the Ubuntu documentation. Some files and commands may be different if
+you use Ubuntu-managed packages.
 
 >**Note**: Ubuntu Utopic 14.10 and 15.04 exist in Docker's `APT` repository but
 are no longer officially supported.
 
 ## Prerequisites
 
-Docker requires a 64-bit installation regardless of your Ubuntu version.
-Additionally, your kernel must be 3.10 at minimum. The latest 3.10 minor version
-or a newer maintained version are also acceptable.
+Docker has two important installation requirements:
 
-Kernels older than 3.10 lack some of the features required to run Docker
-containers. These older versions have known bugs which cause data loss
-and frequently panic under certain conditions.
+- Docker only works on a 64-bit Linux installation.
+- Docker requires version 3.10 or higher of the Linux kernel. Kernels older than
+  3.10 lack some of the features required to run Docker containers and contain
+  known bugs which cause data loss and frequently panic under certain conditions.
 
-To check your current kernel version, open a terminal and use `uname -r` to
-display your kernel version:
+  To check your current kernel version, open a terminal and use `uname -r` to
+  display your kernel version:
 
-    $ uname -r
-    3.11.0-15-generic
-
->**Note**: If you previously installed Docker using `APT`, make sure you update
-your `APT` sources to the new Docker repository.
+  ```bash
+  $ uname -r
+  3.11.0-15-generic
+  ```
 
 ### Update your apt sources
 
```

Comment:
I think by now it's safe to remove the first sentence ("Docker's `APT` repository contains Docker 1.7.1 and higher."). We added it because we switched to a new apt repository (so older versions have to be obtained from the old apt repo)